internal/encrypt/init.go

```diff
@@ -0,0 +1,123 @@
+package encrypt
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/inconshreveable/log15"
+
+	"github.com/sourcegraph/sourcegraph/internal/conf"
+)
+
+var cryptObject Encryptor
+var configuredToEncrypt bool
+var doOnce sync.Once
```

Comment:
To reflect where it is used for:

```suggestion
var initOnce sync.Once
```